Stratis.Bitcoin.Tests/NodeConfiguration/TextFileConfigurationTests.cs

```diff
@@ -0,0 +1,72 @@
+﻿using Stratis.Bitcoin.Configuration;
+using Xunit;
+
+namespace Stratis.Bitcoin.Tests.NodeConfiguration
+{
+    public class TextFileConfigurationTests
```

Comment:
I know the current commenting policy prescribes comments only in main code, not tests, but I would like you to consider commenting the tests as well. Won't hold PR for that though, just for consideration.